scripts/onboarding/managed/disable-monitoring.sh

```diff
@@ -127,7 +127,7 @@ remove_monitoring_tags()
 
   # validate cluster identity for Azure Arc enabled Kubernetes cluster
   if [ "$isArcK8sCluster" = true ] ; then
-   identitytype=$(az resource show -g ${clusterResourceGroup} -n ${clusterName} --resource-type $resourceProvider --query identity.type)
+   identitytype=$(az resource show -g ${clusterResourceGroup} -n ${clusterName} --resource-type $resourceProvider --query identity.type -o json)
```

Comment:
Thanks @nyuen  for making PR on this.  looks like some weird issue with az cli handling quotes.
Can you please sharing the version of the Az CLI and platform you are able to repro this issue? and also, just wanted to let you know that, we will be extension in few extensions and at that point of time, script will not be used any more.

Reply:
Tested with when the default output is configured to **table** as opposed to **json** :
Azure-cli    Azure-cli-core    Azure-cli-telemetry
-----------  ----------------  ---------------------
2.17.1       2.17.1            1.0.6

I've also tested it with the latest version:
Azure-cli    Azure-cli-core    Azure-cli-telemetry
-----------  ----------------  ---------------------
2.20.0       2.20.0            1.0.6


This is the error first error I get in both cases:
cluster identity type: result **-------------- systemassigned**
-e only supported cluster identity is **systemassigned** for Azure Arc enabled Kubernetes cluster type

With az output set to table you get:
az resource show -g ${rgName} -n ${clusterName} --resource-type $resourceProvider --query identity.type
**Result
--------------
SystemAssigned**

with json output you get
az resource show -g ${rgName} -n ${clusterName} --resource-type $resourceProvider --query identity.type -o json
**"SystemAssigned"**

We need to explicitly set the output to json or tsv on every single calls in case the user has a different configuration:
az resource show -g ${rgName} -n ${clusterName} --resource-type $resourceProvider --query identity.type -o tsv
**SystemAssigned**

@ganga1980 thanks for the update regarding the script, makes sense to use the embedded CLI :-)

Reply:
Thanks for this PR and validated, looks good to me.